Editor/AGS.CScript.Compiler/CodeBlockCompiler.cs

```diff
@@ -269,7 +268,7 @@ private void GenerateCodeForExpressionWithoutOperator(Expression expression)
                 {
                     throw new CompilerMessage(ErrorCode.ParentIsNotAStruct, "'" + firstToken.Name + "' is not a struct");
                 }
-                memberName = reader.ReadNextToken();
+                ////Token memberName = reader.ReadNextToken();

```

Comment:
I cannot tell if this is correct. ReadNextToken may advance some iterator, thus commenting its call out may cause further  operations break.

I suggest leaving original variable declaration (above) instead, and doing something like this on line 271:

> reader.ReadNextToken(); // memberName